server/chat.ts

```diff
@@ -733,8 +733,11 @@ export class CommandContext extends MessageContext {
 		}
 		return true;
 	}
+	get willBroadcast() {
```

Comment:
I hate getters. :[

Reply:
Just make it a function.

Reply:
Also I'd probably name it something like `shouldBroadcast()` for clarity?

Reply:
That sounds reasonable. What's wrong with getters though? I'm not attached to this being a getter or anything, just for reference in the future

Reply:
I just prefer code execution to be explicit.